lib/commands/app/rename.ts

```diff
@@ -39,16 +40,19 @@ export default class AppRenameCmd extends Command {
 
 		Note, if the \`newName\` parameter is omitted, it will be
 		prompted for interactively.
-		`;
+	`;
+
 	public static examples = [
 		'$ balena app rename OldName',
 		'$ balena app rename OldName NewName',
+		'$ balena app rename myorg/oldname NewName',
 	];
 
 	public static args: Array<IArg<any>> = [
 		{
-			name: 'name',
-			description: 'application name or numeric ID',
+			name: 'nameOrSlug',
+			description: 'application name or org/name slug',
```

Comment:
Approving this PR, but I think we may want to change this code again as part of PR #2131, re documentation of the meaning of "slug" (even the meaning of "org") and a recommendation of slugs over app names or IDs.

To share more data points, given an org named `οργάνωση` that contains an app named `test33`:
```
$ ./bin/balena app rename οργάνωση/test33 new-test-33
BalenaApplicationNotFound: Application not found: οργάνωση/test33
```

Turns out that the org has handle `_`, and the following succeeds:
```
$ ./bin/balena app rename _/test33 new-test-33
Application _/test33 renamed to new-test-33
```

Further discussion: https://www.flowdock.com/app/rulemotion/i-cli/threads/k9NEA-BfE0IbigEuQL85f0z3mZv